### ISSUE TITLE
Persistent Notifications for Calendar events

### ISSUE THREAD

**Issue by ashokfernandez** [wavebox/waveboxapp#425]
* Which version of Wavebox are you using? [e.g. 3.4.0]
* Which Operating System are you using? [Mac OSX 10.12.6]

**Comment by Thomas101** [comment]
Yeah, I think this would make a nice addition. I think there may be some support in the underlying notification library so I'll take a look what we can do.

In the meantime, if you've enabled desktop notifications rather than interruptive alerts in your Google Calendar settings it does produce a red dot indicator above the calendar icon. Not as great as having the notification stick around, but might help

**Comment by liquidonline** [comment]
I'd like to add it would be nice for the linux build too.  I'm using KDE plasma 5.11.x via KDE Neon distribution, and I *often* miss calendar notifications.  Chrome for linux now has (experimental) support for native OS notifications, too, and I find they do a good job of notifying - maybe you can take a look and imitate?
